pkg/client-sdk/store/kv/vtxo_repository.go

```diff
@@ -58,7 +58,9 @@ func (s *vtxoStore) AddVtxos(_ context.Context, vtxos []types.Vtxo) (int, error)
 	return len(addedVtxos), nil
 }
 
-func (s *vtxoStore) SpendVtxos(ctx context.Context, outpoints []types.VtxoKey, spentBy string) (int, error) {
+func (s *vtxoStore) SpendVtxos(
+	ctx context.Context, outpoints []types.Outpoint, spentByMap map[string]string, settledBy string,
```

Comment:
we may use Outpoint struct as map key instead of string ? to make it easier to understand what's inside and avoid string serialisation

Reply:
done